region/crateria/central.json

```diff
@@ -2634,10 +2634,10 @@
                       "mode": "any",
                       "artificialMorph": true
                     }},
-		    "h_canArtificialMorphMovement"
+		                "h_canArtificialMorphMovement"
```

Comment:
I'm not sure what's going on with the weird spacing. I think it has something to do with tabs counting as a different number of spaces, which only happens in Crateria Central ... I wonder if there is a way we can fix it? On my end it looked better before.

Reply:
It's looks better the old way on github now to me too.  
These lines did have some tabs used to indent.  Everywhere else only uses spaces.  I'll look into it some more.
It lines up in MVS if we count each tab as 8 spaces.  This line turns 2tabs into16spaces and is 12spaces too many.

Reply:
I removed all the tabs and it looks ok to me on site and in IDE.  It's possible I added spaces before so it looked ok to me but left the tabs and they were displaying differently on the site.